src/nupic/encoders/base.py

```diff
@@ -70,8 +86,8 @@ class Encoder(object):
      subclasses:
 
   - :func:`~nupic.encoders.base.Encoder.getDecoderOutputFieldTypes` - must be
-     implemented by leaf encoders. Returns :class:`nupic.data.fieldmeta.FieldMetaType`.XXXXX
-     (e.g., :class:`nupic.data.fieldmeta.FieldMetaType`.float)
+     implemented by leaf encoders. Returns :class:`~nupic.data.field_meta.FieldMetaType`.XXXXX
+     (e.g., :class:`~nupic.data.field_meta.FieldMetaType`.float)
   - :func:`~nupic.encoders.base.Encoder.getWidth` - returns the output width, in
      bits
```

Comment:
Consider using a backslash to indicate that the line continues. It escapes the newline character instead of breaking the block in the HTML rendering. Examples:

Before:
<img width="718" alt="screen shot 2017-05-02 at 4 14 39 pm" src="https://cloud.githubusercontent.com/assets/2488251/25643111/9f1d034e-2f52-11e7-8a47-3ff95327f80a.png">


After:
<img width="735" alt="screen shot 2017-05-02 at 4 16 19 pm" src="https://cloud.githubusercontent.com/assets/2488251/25643127/b4131658-2f52-11e7-85bf-e3daad396fbc.png">


Edits in `nupic/encoders/base.py` to fix the newline issue (I just added trailing `\`'s where needed):

```python
class Encoder(object):
  """
  An encoder converts a value to a sparse distributed representation.

  This is the base class for encoders that are compatible with the OPF. The OPF
  requires that values can be represented as a scalar value for use in places
  like the SDR Classifier.

  .. note:: The Encoder superclass implements:

  - :func:`~nupic.encoders.base.Encoder.encode` - returns a numpy array \
    encoding the input; syntactic sugar on top of encodeIntoArray. If pprint, \
    prints the encoding to the terminal.
  - :func:`~nupic.encoders.base.Encoder.pprintHeader` - prints a header \
    describing the encoding to the terminal.
  - :func:`~nupic.encoders.base.Encoder.pprint` - prints an encoding to the \
    terminal.

  .. warning:: The following methods and properties must be implemented by \
     subclasses:

  - :func:`~nupic.encoders.base.Encoder.getDecoderOutputFieldTypes` - must be \
     implemented by leaf encoders. \
     Returns :class:`~nupic.data.field_meta.FieldMetaType`.XXXXX \
     (e.g., :class:`~nupic.data.field_meta.FieldMetaType`.float)
  - :func:`~nupic.encoders.base.Encoder.getWidth` - returns the output width, \
     in bits.
  - :func:`~nupic.encoders.base.Encoder.encodeIntoArray` - encodes input and \
     puts the encoded value into the numpy output array, which is a 1-D array \
     of length returned by :func:`~nupic.encoders.base.Encoder.getWidth`.
  - :func:`~nupic.encoders.base.Encoder.getDescription` - returns a list of \
     (name, offset) pairs describing the encoded output.
  """

```

Ref of a similar issue for another block type: http://stackoverflow.com/a/28328736